### DISCUSSION TITLE
Photo during timelapse?

### DISCUSSION THREAD

**Discussion by cfinke** [gopro/labs#534] [Q&A] [answered]
I'd like to do daily sunup to sundown timelapses but also have the camera take a full-size photo every hour so I can stitch together a year-long timelapse at the end of the year. I've read through a lot of documentation and I don't think this is possible without doing something like having the camera make a separate timelapse for every hour and take a photo inbetween, but does anyone have any ideas?

**Comment by dnewman-gpsw** [comment, marked as answer]
Easier to script a tool like FFMPEG to extract a frame from timelapse video.  Or use TL Photo modes.  But stopping and starting a timelapse, will introduce potential exposure jumps when joining back together.
